src/aac/spec/spec.yaml

```diff
@@ -178,19 +176,19 @@ data:
 data:
   name: Step
   fields:
-    - name: step  # this is the human readable narrative suitable for document generation
+    - name: step
       type: string
-    - name: source  # this should be a name of a participant (i.e. modelled system component)
+    - name: source
       type: string
-    - name: target  # this should be a name of a participant (i.e. modelled system component)
+    - name: target
       type: string
-    - name: action  # this as a behavior reference on the target or a subordinate use case definition
+    - name: action
       type: string
-    - name: if      # would create an alt in a sequence diagram
```

Comment:
I think these comments provide valuable context, why delete them?

Reply:
I don't really see what valuable context this provides, honestly. That said, if we chose to keep this for documenting the spec, we should probably add documentation to the entire spec. To that end, maybe we put the description key as an optional part of `Field` so that this kind of information can be provided and used in generation, etc.

Reply:
Well, I have already done that in GenPlugin so it's doable now. 

https://github.com/jondavid-black/AaC/blob/2445628738d69188d4f6aafdd0482500209ecf89/src/aac/genplug.py#L87-L94

Reply:
That's where I got the idea. 😉

Reply:
Resolved with #136